ArticleTemplates/assets/scss/modules/_quiz.scss

```diff
@@ -424,5 +440,72 @@
                 }
             }
         }
+
+        // Quiz Results
+        .quiz-results {
+            .quiz-results {
+                &__description {
```

Comment:
When I first worked here, I wrote my scss the same, however I was convinced to refactor and write out the classes without splitting in the middle as it made grepping the codebase a lot easier when looking for a specific class. Just a thought.